src/cards/index.tsx

```diff
@@ -57,7 +57,7 @@ const Cards = React.forwardRef(function <T = any>(
   }: CardsProps<T>,
   ref: React.Ref<CardsProps.Ref>
 ) {
-  const { __internalRootRef } = useBaseComponent('Cards');
+  const { __internalRootRef } = useBaseComponent('Cards', { loading, selectedItems, selectionType, items, trackBy });
```

Comment:
Same question about serialization friendliness here

`selectedItems` and `items` are nested and possibly even recursive objects
`trackBy` can be a function